datalad_registry/tasks/__init__.py

```diff
@@ -631,3 +636,101 @@ def chk_url_to_update(
         db.session.commit()
 
     return ChkUrlStatus.OK_UPDATED if is_record_updated else ChkUrlStatus.OK_CHK_ONLY
+
+
+class FailedSubmission(TypedDict):
```

Comment:
why `TypedDict` and not `dataclass`es which seems to be more common construct pattern to use?

Reply:
The reason I used type `TypedDict` is because a dictionary displays better in the Flower interface.